pkgs/development/libraries/gtkd/default.nix

```diff
@@ -84,6 +84,9 @@ stdenv.mkDerivation rec {
 
   installFlags = "prefix=$(out)";
 
+  # Workaround for https://github.com/NixOS/nixpkgs/issues/40397
```

Comment:
Can you add a comment here that this should be removed after upgrading binutils to 2.31?

Reply:
Done